schemas/resources/schemas/visualization/v1.1.0/validate-test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+#set -e
+
+source ~/gibs/tool/init.sh
```

Comment:
I know these were in the schmea directory. I don't see an issue with having them in here but, is there a reason they **should** be in here?